common/src/main/java/com/wynntils/features/user/WorldWaypointDistanceFeature.java

```diff
@@ -83,6 +84,10 @@ public void onRenderLevelPost(RenderLevelEvent.Post event) {
         float dy = (float) (location.y + 0.5 - cameraPos.y);
         float dz = (float) (location.z + 0.5 - cameraPos.z);
 
+        if (location.y <= 0 || location.y > 255) {
+            dy = 0;
```

Comment:
This is a bit weird IMO. What if we clamped the value between 0 and 255? So basically, if the value was -10, we get 0, but if it was 300, we get 255? Isn't that better? (Use `MathUtils#clamp` if you do so)

Reply:
clamping the value will still result the indicator render in mostly inaccessible locations, the aim of it is to reduce the instance where indicator show up at confusing location, like underground or way high up in the sky, for player placed waypoint and LablePoi respectively

Reply:
But, y level zero means we still render underground?

Reply:
Or does 0 here mean we render at camera level?

Reply:
0 is the delta y, which is the difference between the location's y level and camera's y level, and then it's used to convert the worldspace location to it's screenspace representative

Reply:
and that's the reason in first iteration I took the obvious approach and replace location.y with cameraPos.y without realizing

Reply:
> Or does 0 here mean we render at camera level?

so this is correct

Reply:
I see. Okay.